app/models/dialog.rb

```diff
@@ -13,6 +13,7 @@ class Dialog < ApplicationRecord
 
   before_destroy :reject_if_has_resource_actions
   validates      :name, :unique_within_region => true
+  validates      :dialog_tabs, :presence => {:message => ->(object, _) { "missing for Dialog #{object.name}" }}
```

Comment:
this is removing the localization here.
Pretty sure this will go to the ui. Not sure if we want to change this.

Can we introduce a `validates` but use localization?

something like:
```ruby
validates :dialog_tabs, :presence => {:message => ->(object, _) { _("Dialog %{dialog_label} must have at least one Tab") % {:dialog_label => label} } 
```